.github/CHANGELOG.md

```diff
@@ -508,6 +508,70 @@
 * Due to the addition of `density_matrix()` as a return type from a QNode, tuples are now supported by the `output_dim` parameter in `qnn.KerasLayer`.
   [(#1070)](https://github.com/PennyLaneAI/pennylane/pull/1070)
 
+* Added functionality for constructing and manipulating the Pauli group
+  [(#1159)](https://github.com/PennyLaneAI/pennylane/pull/1159).
+  For example, we can iterate through the 3-qubit Pauli group like so:
+
+  ```python
+  from pennylane.grouping.pauli_group import pauli_group_generator
+
+  for pauli in pauli_group_generator(3):
+      print(pauli)
+  ```
+
+  We can also construct and store the full group, and multiply together
+  its members at the level of Pauli words using the `pauli_mult` and
+  `pauli_multi_with_phase` functions. This can be done on
+   arbitrarily-labeled wires as well, by defining a wire map.
+
+  ```python
+  from pennylane.grouping.pauli_group import pauli_group, pauli_mult
```

Comment:
is `pauli_group` the same as `pauli_group_generator`, but it creates a list rather than a generator?

I wonder if it makes sense just to have one function (`pauli_group`) that always returns a generator -- it is easy to convert it to a sequence if needed

Reply:
Yes that seems reasonable - I was thinking mostly that it might be confusing if people hadn't encountered generators like this before, but we can just make it clear with examples/documentation :)